Yauheni_Kastsevich/05.Ansible.start/vars/groups.yaml

```diff
@@ -0,0 +1,12 @@
+mygroups:
+  - name: managers
+  - name: engineers
+
+users:
```

Comment:
please keep this users's  data in ansible vault

Reply:
done